backend/frontend/tests/test_course_component_details_pages.py

```diff
@@ -69,7 +69,58 @@ def test_link_project_page_displays_correct_details(self):
         self.assertIn("Feb. 13, 2024, 2:06 p.m.", body)
         self.assertIn("learner_reviewer@umuzi.org", body)
         self.assertIn(
-            "https://raw.githubusercontent.com/Umuzi-org/tech-department/master/content/projects/tdd/simple-calculator/part-1/_index.md",
+            self.recruit_project.content_url,
             body,
         )
         self.assertIn("No link submitted yet", body)
+
+    def test_link_submission_form_correctly_updates_link_submission_after_link_update(
```

Comment:
change `test_link_submission_form_correctly_updates_link_submission_after_link_update` to `test_link_submission_form_correctly_updates_link_submission` or make it less confusing